game_board/src/fr/lecampusnumerique/cda2025/javaalgo/boardgames/PlayerTurns/PlayerTurns.java

```diff
@@ -0,0 +1,14 @@
+package fr.lecampusnumerique.cda2025.javaalgo.boardgames.PlayerTurns;
+
+import fr.lecampusnumerique.cda2025.javaalgo.boardgames.players.Player;
+
+import java.util.Scanner;
+
+public interface PlayerTurns {
+    Player switchPlayer();
+
+    boolean checkMove();
```

Comment:
in order to fit the way we name our boolean methods and attributs, how could we rename this one?

getIsMoveCompatible ?